scopes/compositions/compositions/compositions.module.scss

```diff
@@ -81,13 +81,11 @@ $panelBg: #fafafa;
 
 .noCompositionsPage {
   padding: 50px 40px 100px 40px;
-  width: auto;
+  width: 100%;
```

Comment:
width: 100%? it seems strange. I guess it's because of a flex parent.
maybe we can set `.$parent > * { width: 100%; }` `$parent > * { flex: 1 1; }` ?

Reply:
ok